tests/ci/sync-to-aws.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -euo pipefail
+
+source /tmp/.env
+
+[ -z "$IMAGE_KEY" ] && error "Could not find '$IMAGE_KEY', qcow image has not been created"
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
+S3_BUCKET_NAME="fedora-testing-farm-image-import"
+AWS_CLI="aws"
```

Comment:
Is `aws` installed in the CI machine?
If we need it, we should add it to the list of packages we install, to be sure:
https://github.com/osbuild/automotive-ci/blob/main/tests/ci/image-build.fmf#L22

Reply:
Or, consider using the aws cli v2 which is based on container image.

Reply:
`aws` needs to be installed only on the bare-metal machine

Reply:
We should add a section where we provision the target environment (bare metal machine)

Reply:
@lioramilbaum there is a section for that, the one I linked in the first comment.

There is also the `setup.sh` for more complex steps. All those files are executed via `tmt` at the bare-metal machine and that's why we need to install and configure the `aws` there and not at the container.

Reply:
thanks @juanje I added it, my only concern is that this package is downloaded from an external url (https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html#install-linux-bundled-sudo) and it might not be found in the default repositories of a server.